src/coreclr/inc/clrconfigvalues.h

```diff
@@ -259,7 +259,7 @@ RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_legacyCorruptedStateExceptionsPolicy, W("le
 CONFIG_DWORD_INFO(INTERNAL_SuppressLostExceptionTypeAssert, W("SuppressLostExceptionTypeAssert"), 0, "")
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_UseEntryPointFilter, W("UseEntryPointFilter"), 0, "")
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_Corhost_Swallow_Uncaught_Exceptions, W("Corhost_Swallow_Uncaught_Exceptions"), 0, "")
-RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableNewExceptionHandling, W("EnableNewExceptionHandling"), 0, "Enable new exception handling.");
+RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableNewExceptionHandling, W("EnableNewExceptionHandling"), 1, "Enable new exception handling.");
```

Comment:
is there a way to configure this via runtimeconfig as well? We had some asks for having even W^X configurable through that as well.

Reply:
No, let me figure out how to do that.

Reply:
@mangod9  I've just added a commit that adds "System.Runtime.EnableNewExceptionHandling" setting that can control that from runtime.config.

Reply:
Could we not call this "New" exception handling? I've worked on way too many features where the "New" thing was 15 years old and its name confused everyone at all times.

Reply:
Ah, I suppose thats the name throughout the product at this point. Oh well. I suppose New it is. Sigh.

Reply:
@davidwrighton yeah, I know. I couldn't come up with anything better over the time. If you have a better idea, I'd be happy to rename it.

Reply:
Just a quick idea - would using `DOTNET_ExceptionHandlingMode` and `System.Runtime.ExceptionHandlingMode` with a numeric value feel better? We would use 0 for the old EH, 1 for the new EH and if we ever moved it to something else, then we could keep incrementing it.

Reply:
> I couldn't come up with anything better over the time. 

A better name would be `LegacyExceptionHandling` with opposite meaning. It makes it clear that it is something that one should not use and that is likely going to disappear at some point.